Frontend/ADS_UI/src/app/models/size.ts

```diff
@@ -0,0 +1,5 @@
+export enum Size{
+    s=0,
+    m=1,
+    l=2
```

Comment:
Not sure, but is it necessary to specidy numbers?

Reply:
it is not necessary, but it is not bad practise, i guess